diagnostic.md

```diff
@@ -98,11 +117,19 @@ JSON.
 ```
 
 ```sh
-# your answer here
-```
+curl --include --request PATCH http://localhost:3000/books/$42```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = $.ajax({
+            type: "PATCH", 
+            dataType: 'json',
+            url: "http://example.com/data/donuts",
+            async: false,
+            contentType: "data/donuts; charset=utf-8",
+            success: function (msg) {
+                console.log(msg);
+            }
+ });;
```

Comment:
```js
let createDonut = function (donutID) {
  return $.ajax({
    url: 'http://localhost:3000/donuts/' + donutID,
    method: 'PATCH',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```

Again, not sure where that came from. Let me know if this doesn't make sense after looking at it.